### ISSUE TITLE
Dropdown ARIA issues.

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#4676]
The VP of Engineering and VSO (Makers of JAWS) reported these issues. They look like good suggestions.
Here's the text:

•	Live Regions are used to speak the selected item. This is nonstandard behavior and is not necessary when ARIA markup is correct. It also leads to double speaking in many cases.
•	When the dropdown is in the opened state, focus is on an item of role Listbox. It should instead be placed on an item with role=option  with a parent item of role=listbox.
•	When the dropdown is in the collapsed state, it’s given a role of combobox, but since the collapsed thing doesn’t have edit semantics, there’s no way for a screen reader user when in the virtual buffer to obtain the text of the selected item. 

We believe the example at:
https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/listbox/listbox-collapsible.html is the proper ARIA implementation. It works well with JAWS in IE, Firefox and Chrome and I expect it works well with NVDA as well. This presentation is what users have come to expect.

Please consider altering the fabric controls to function in the same way as the example. We’d appreciate it if you reach out to us to discuss if you have concerns with this recommendation.

**Comment by wygoodin** [comment]
I've started work on moving towards the way it's being done in the given w3 link.

**Comment by cliffkoh** [comment]
Fixed in #5385.
